pkg/envoy/resources.go

```diff
@@ -174,8 +187,8 @@ func (cache *NPHDSCache) handleIPDelete(npHost *envoyAPI.NetworkPolicyHosts, pee
 
 	// If removing this host would result in empty list, delete it.
 	// Otherwise, update to a list that doesn't contain the target IP
-	if len(npHost.HostAddresses) <= 1 {
-		cache.Delete(NetworkPolicyHostsTypeURL, peerIdentity)
+	if len(npHost.HostAddresses) == 0 {
```

Comment:
Before this conditional would match when HostAddresses had 1 entry, in addition to 0. What's the reason  for this change?

Reply:
Oops, this was unintentional change, I was reading it like `<= 0` which made no sense. Thanks for spotting this!